server/tests/testarea.py

```diff
@@ -0,0 +1,39 @@
+from gqlComms import listHistograms, createHistogram, deleteHistogram,getHistogram,getHistograms
+import numpy as np
+import datetime
+import argparse
+
+def test_createHist():
+    NUM = 100
+    LENGTH = 50
+    LOW = 0
+    HIGH =1000
+    rng = np.random.default_rng()
+    now = datetime.datetime.now()
+    runHeader = now.strftime("%Y%m%d_run")
+
+    params ={
+            'id': 100,
+            'x': np.arange(LENGTH).tolist(),
+            'y': rng.integers(low=LOW, high=HIGH, size=LENGTH).tolist(), 
+            'name': f'{runHeader}{id}', 
+            'type': 'static_test', 
+            'isLive': False,
+        }
+    createHistogram( **params )
+    assert getHistogram(100)
```

Comment:
Unfortunately I don't think it's allowed to simply assert this function, because a graphql query returns response['data'] if successful and response['error'] if something goes wrong. Both of which are technically valid results

Assert should be a clear boolean result if possible, e.g. `getattr(response, 'error', None) is None`

Reply:
I should add that it is also possible to make the `check_response_errors` raise an error instead of print to stdout, which would make your `assert function()` idea more plausible. But I still think you should validate the returned output itself